pyproject.toml

```diff
@@ -24,6 +24,13 @@ dev = [
     "ipython>=8.18.1",
 ]
 
+test = [
```

Comment:
Unintended additions?

Reply:
Yes. Kinda. Originally on purpose, but I decided it's cleaner to switch between .venvs in libs than try to manage one. So I've gottten rid of it. :)